README.md

```diff
@@ -139,12 +139,16 @@ Participation in Argo Workflows is governed by the [CNCF Code of Conduct](https:
 * [Argo Workflows and Pipelines - CI/CD, Machine Learning, and Other Kubernetes Workflows](https://youtu.be/UMaivwrAyTA)
 * [Argo Ansible role: Provisioning Argo Workflows on OpenShift](https://medium.com/@marekermk/provisioning-argo-on-openshift-with-ansible-and-kustomize-340a1fda8b50)
 * [Argo Workflows vs Apache Airflow](http://bit.ly/30YNIvT)
+* [Beyond Prototypes: Production-Ready ML Systems with Metaflow and Argo](https://github.com/terrytangyuan/public-talks/tree/main/talks/kubecon-na-2023-metaflow-argo)
```

Comment:
It might be good to make these permalinks instead of links to `main`; that way even if you rename, change directory structures, etc, the links still work. 

Same with why we use permalinks to code and to the docs.

Although if you plan to update these talks, maybe `main` is fine, but idk if you do